.gitignore

```diff
@@ -17,3 +17,4 @@ java/bench/data
 subprojects/*
 !subprojects/packagefiles
 !subprojects/*.wrap
+.vscode
```

Comment:
Thank you, but please revert this change, @HuaHuaY . We don't want to have a hidden change under the title, `ORC-2028: [C++] Fix ReadRangeCache uses freed memory`. I'd like to recommend to open another PR for this.